.github/workflows/ci.yaml

```
@@ -0,0 +1,28 @@
+name: ci
+
+on:
+  pull_request:
+  push:
+
```

Let's limit the push events to just pushes to `master`:

```suggestion
  push:
    branches:
      - master
```